enumcover.go

```diff
@@ -87,11 +88,20 @@ func checkConsts(pass *analysis.Pass, n ast.Node, typeName string) {
 									if lit, ok := value.(*ast.BasicLit); ok {
 										namesForType[unquote(lit.Value)] = true
 									}
+									if lit, ok := value.(*ast.Ident); ok {
+										namesForType[lit.Name] = true
+									}
 								}
 							}
 						}
 					}
+
 					namesForType[n.Name] = true
+					if n.Obj == nil {
+						//store the value
+						val := constMap[n.Name]
+						namesForType[val] = true
```

Comment:
I don't think I understand this bit. Can you give an example of some code where n.Obj would be nil and we should be assuming the node's name is the value in this constMap?